lib/util/git/error.ts

```diff
@@ -124,7 +124,7 @@ export function handleCommitError(
     logger.info(
       'Branch update was rejected because local copy is not up-to-date.'
     );
-    return null;
+    throw new ExternalHostError(err, 'git');
```

Comment:
`'[rejected] (stale info)'` can be added to `externalHostFailureStrings` instead, but log message would be too verbose IMO.

Anyway, please suggest better alternative if the solution is okay in general.

Reply:
How does it look in the logs with your current suggestion?

Reply:
> How does it look in the logs with your current suggestion?

Please, take a look here: https://github.com/renovatebot/renovate/pull/14228#issuecomment-1039875472